docs/current/guides/757394-use-services.md

```diff
@@ -0,0 +1,326 @@
+---
+slug: /757394/use-services
+displayed_sidebar: "current"
+category: "guides"
+tags: ["go", "python", "nodejs"]
+authors: ["Alex Suraci"]
+date: "2023-03-07"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Embed from '@site/src/components/atoms/embed.js'
+
+# Use Services in Dagger
+
```

Comment:
@vikram-dagger It would be good to include a note so people know how to opt out, just in case.

Something like this, but the formatting could be better:

```suggestion
:::tip
The new services feature changes Dagger's entire network stack. If something seems to have broken, you can disable it by setting the following environment variable prior to running the engine: `export _EXPERIMENTAL_DAGGER_SERVICES_DNS=0`

Please report an issue if this is the case!
:::
```

Reply:
Added

Reply:
The _note_ admonition is easy to miss. May I suggest using a _tip_ or _caution_?